portfolio/src/main/webapp/index.html

```diff
@@ -1,52 +1,84 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="https://kit.fontawesome.com/908160b183.js" crossorigin="anonymous"></script>
-    <script src="script.js"></script>
-  </head>
-
-  <body>
-      <header class="header">
-        <div class="wrapper">
-          <nav class="main-nav">
-              <ul>
-                  <li><a href="#">Home</a></li>
-                  <li><a href="#">About me</a></li>
-                  <li><a href="#">Portfolio</a></li>
-                  <li><a href="#">Thoughts</a></li>
-                  <li><a href="#">Contact me</a></li>
-              </ul>
-          </nav>
-          <img src="https://media.istockphoto.com/vectors/cute-hanging-sloth-vector-id1093750176?k=20&m=1093750176&s=612x612&w=0&h=c9jgY50j2Kzj6y23dMTypnlPpa-TkevMNObLz7e94MQ=" alt="photo of me" class="profilepic">
-        </div>
-      </header>
-
-      <main>
-
-            <p class="selfintro">Hi, I'm Bella Li! I am an undergraduate student at Emory Univeristy studying Computer Science and Philosophy.</p>
-
-      </main>
-
-      <footer class="footer">
-        <div class="wrapper">
-          <p class="endnote">Great to meet you! Please follow me on social media so we can keep in touch. Cheers!</p>
-          <ul>
-              <li><a href="#" class="INSTA"><i class="fa-brands fa-instagram-square"></i></a></li>
-              <li><a href="#" class="LINK"><i class="fa-brands fa-linkedin"></i></a></li>
-              <li><a href="#" class="GIT"><i class="fa-brands fa-github-square"></i></a></li>
-              <li><a href="#" class="FB"><i class="fa-brands fa-facebook-square"></i></a></li>
-          </ul>
-          <form action="/form-handler" method="POST" class="submit">
-            <p>Enter some text here:</p>
-            <textarea name="text-input">hello world</textarea>
-            <br/>
-            <input type="submit" />
-          </form>
-          <p class="copyright"></p>
-        </div>
-      </footer>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/908160b183.js" crossorigin="anonymous"></script>
+  <script src="script.js"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Anek+Latin&family=Raleway:wght@350;400&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wdth@8..144,100;8..144,130&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+  <header class="header">
+    <div class="wrapper">
```

Comment:
Single <div> nested in header suggests that the two elements could be collapsed into one with their styles combined.